cli/fathom_web/commands/vectorize.py

```diff
@@ -41,11 +41,13 @@ def main(ruleset_file, samples_directory, fathom_fox_dir, fathom_trainees_dir, o
     """
     Create feature vectors for a directory of training samples using a Fathom Ruleset.
 
-    RULESET_FILE: Path to the ruleset.js file
+    \b
```

Comment:
Huh; never seen that before. How is \b different from \n? They seem to behave the same when I swap them.

Reply:
\b will prevent line wrapping for the entire paragraph: https://click.palletsprojects.com/en/7.x/documentation/#preventing-rewrapping. For \n to work the same, I'd have to put it on many more lines.